### ISSUE TITLE
issue: Module parse failed: Unexpected token (74:61) You may need an appropriate loader to handle this file type

### ISSUE THREAD

**Issue by xiaozhangclassmater** [puppeteer/puppeteer#8822]
### Bug description

Steps to reproduce the problem:

1. …
I have downloaded the 16.2 Puppeteer but I am using it in Vue and Webpack tells me it is missing loader, what should I do?

### Puppeteer version

16.2.0

### Node.js version

16.14.0

### npm version

8.5.1

### What operating system are you seeing the problem on?

Windows

### Relevant log output

```shell
I have downloaded the 16.2 Puppeteer but I am using it in Vue and Webpack tells me it is missing loader, what should I do?
```

**Comment by OrKoN** [comment]
Please provide a reproducible minimal example? From the description it sounds like your project might require additional configuration which is probably unrelated to Puppeteer.

**Comment by Luckqingyouzhang** [comment]
I use it in vue2. I install it according to the configuration of the official website, but I can't use it. The error message says that there is no loader to load it. Can you help me? thank you。

![image](https://user-images.githubusercontent.com/94174307/185855981-6eba632a-17e7-4e94-bb3a-b479078df161.png)

**Comment by OrKoN** [comment]
I'd recommend reaching out the Vue/Webpack community (e.g., via the Stackoverflow) to see what the problem might be. It does not look like a problem with Puppeteer itself.

**Comment by Ans1998** [comment]
The lower version can be fixed
npm install puppeteer-core@18.1.0

**Comment by xiaozhangclassmater** [comment]
> The lower version can be fixed npm install puppeteer-core@18.1.0

Yes, the lower version will not have this error
